hazelcast/src/main/java/com/hazelcast/jet/impl/execution/init/JetInitDataSerializerHook.java

```diff
@@ -212,8 +213,8 @@ public IdentifiedDataSerializable create(int typeId) {
                     return new NoopP.NoopPSupplier();
                 case CHECK_LIGHT_JOBS_OP:
                     return new CheckLightJobsOperation();
-                case SQL_SUMMARY:
```

Comment:
Is it OK to re-use the class ids like this? I know Jet does not have backward compatibility but I remember(not %100 sure) we have some sort of RU guarantee for SQL. Can this be a problem in this case?

Reply:
It's OK, this class was never returned to the client. It broke 5.1 release verification, so we did a quick fix not to use it.

Reply:
What about member-to-member communication? Can we send/receive this class from 5.1 members during the rolling upgrade?

Reply:
No, that class is not used in any communication in 5.1.